src/sprites/RigidBody.lua

```diff
@@ -88,7 +88,16 @@ function RigidBody:update()
 
   -- incorporate gravity
 
-  self.velocity = self.velocity + (gmt.vector2D.new(0, 1) * _G.delta_time) * self.g_mult
+  if not groundFound then
+    -- Adds gravity vector to current velocity
+
+    self.velocity = self.velocity + (gmt.vector2D.new(0, 1) * _G.delta_time) * self.g_mult
```

Comment:
@CMorooney if we want to cap max fall speed let's add the following lines in here. Something like this:

if self.velocity.y > [maxFallSpeed] then 
    self.velocity.y = maxFallSpeed
end